### PR TITLE
fix #80069 greenweez.com

### LINKED ISSUE #80069: www.greenweez.com
### Issue URL (Incorrect Blocking)
 [https://www.greenweez.com/la-compagnie-du-marron-farine-de-chataigne-500g-p112625?term=farine+ch%C3%A2taigne](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.greenweez.com%2Fla-compagnie-du-marron-farine-de-chataigne-500g-p112625%3Fterm%3Dfarine%2Bch%25C3%25A2taigne)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard/AdGuard Pro for iOS v4.1.1
Browser: | Safari
DNS filtering: | disabled
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard French

### PR DESCRIPTION
**Pull request by wout-codes**
#80069

### CODE REVIEW

File: AnnoyancesFilter/sections/cookies_whitelist.txt [hunk 1]
```diff
@@ -2,6 +2,11 @@
 !######### White list. Fixing filtering errors #########
 !#######################################################
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/80069
+!#if (adguard_app_ios || adguard_ext_safari || adguard_ext_android_cb)
+greenweez.com#@##staticBackdrop
```
Comment (BlazDT) [inline AnnoyancesFilter/sections/cookies_whitelist.txt:7]: No need for whitelist. Just change existing rule `greenweez.com###staticBackdrop` to `greenweez.com#$##staticBackdrop { display: none !important; }`

**Review by BlazDT — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/cookies_whitelist.txt (hunk 1 above)
Comment (wout-codes) [reply AnnoyancesFilter/sections/cookies_whitelist.txt:7]: We are both right in different perspective. I thought in a different way and you thought in a different way. We should different thoughts ;)

Comment (BlazDT) [reply AnnoyancesFilter/sections/cookies_whitelist.txt:7]: Yours is one more rule.

Comment (wout-codes) [reply AnnoyancesFilter/sections/cookies_whitelist.txt]: @BlazDT Fixed. Please take a look.

**Review by BlazDT — APPROVED** [review]

**Pull request merged**